### PR TITLE
Added Activity controller + /activity endpoint

### PR DESCRIPTION
**Pull request by mickeyn**
This new endpoint will replace the query currently generated and
sent from WEB.

### CODE REVIEW

File: lib/MetaCPAN/Document/Release.pm [hunk 1]
```diff
@@ -591,5 +592,77 @@ sub requires {
     };
 }
 
+sub _activity_filters {
+    my ( $self, $params, $start ) = @_;
+    my ( $author, $distribution, $module, $f )
```
Comment (oalders) [inline lib/MetaCPAN/Document/Release.pm]: Could we clarify what `$f` is?  It's not clear to me from the name or the context of the code.  :)

File: lib/MetaCPAN/Document/Release.pm [hunk 2]
```diff
@@ -591,5 +592,77 @@ sub requires {
...
+        = @{$params}{qw( author distribution module f )};
+
+    my @filters
+        = ( { range => { date => { from => $start->epoch . '000' } } } );
```
Comment (oalders) [inline lib/MetaCPAN/Document/Release.pm]: Could we add a comment as to what the `000` suffix does?

File: lib/MetaCPAN/Document/Release.pm (hunk 1 above)
Comment (mickeyn) [inline lib/MetaCPAN/Document/Release.pm]: this is from the original code in WEB, dating back to 2012, i'll rename to `$new_dists`

File: lib/MetaCPAN/Document/Release.pm (hunk 2 above)
Comment (mickeyn) [inline lib/MetaCPAN/Document/Release.pm]: this is from the original code in WEB, dating back to 2011 - first commit to the activity controller... of course no explanation there :)

Comment (oalders) [inline lib/MetaCPAN/Document/Release.pm]: Heh, ok.  :)

**Pull request merged**